python_modules/dagster/dagster/_daemon/auto_run_reexecution/auto_run_reexecution.py

```diff
@@ -152,8 +139,27 @@ def retry_run(
         )
     )
 
-    strategy = get_reexecution_strategy(failed_run, instance) or DEFAULT_REEXECUTION_POLICY
+    _, run_group = check.not_none(instance.get_run_group(failed_run.run_id))
+    run_group_list = list(run_group)
+
+    # it is possible for the daemon to die between creating the run and submitting it. We account for this
+    # possibility by checking if the a run already exists in the run group with the parent run id of the
+    # failed run and resubmit it if necessary.
+    existing_retried_run = get_retried_run_if_exists(failed_run, run_group_list)
+    if existing_retried_run is not None:
+        # ensure the failed_run has the AUTO_RETRY_RUN_ID_TAG set
+        if failed_run.tags.get(AUTO_RETRY_RUN_ID_TAG) is None:
```

Comment:
[1]

There is an issue with adding the auto-retry tag here since this would also capture manual retries that are launched before the daemon processes this run for a manual retry

Reply:
example:
run A fails
manual retry of run A `A_m` is launched
retry daemon processes run A 

with the current daemon behavior, no new run would be created, basically considering the manual retry as the retry that would have been launched

with this PR, we would maintain that behavior, but we still want a way to mark on run A that the retry has been launched (ie that it's not waiting for a retry to get launched). But setting the id of the manual run as `auto_retry_run_id` feels wrong